workers/unity/Packages/com.improbable.gdk.core/Utility/EntityBuilder.cs

```diff
@@ -88,6 +129,12 @@ public EntityBuilder AddMetadata(string metadata, string writeAccess)
             return AddComponent(new ComponentData(schemaData), writeAccess);
         }
 
+        /// <summary>
+        ///     Sets the Read ACL for an entity.
+        /// </summary>
+        /// <param name="attribute">The worker attribute which has read access over this entity.</param>
```

Comment:
Possibly not for this pr by why does this have two arguments.
`params string[]` would do

Reply:
Makes the xml docs look weird too

Reply:
Some people expressed a preference for the `List<string>` one iirc

Reply:
Its to enforce that you add at least one I think

Reply:
Yeah, that was my clumsy attempt to enforce that at compile time, rather than run time.